scripts/install/opt/seagate/cortx/motr/bin/motr_mini_prov.py

```diff
@@ -664,33 +665,27 @@ def config_logger(self):
                        )
     return logger
 
-def clean_ivt_data(self):
-    if os.path.exists(MOTR_LOG_DIR):
-        self.logger.info("Removing addb directories")
-        dnames_addb = []
-        pattern="{}/**/addb*".format(MOTR_LOG_DIR)
-        for dname in glob.glob(pattern, recursive=True):
-            dnames_addb.append(dname)
+def remove_dirs(self, log_dir, patterns):
+    if len(patterns) == 0:
```

Comment:
There is no validation of log_dir here. This is dangerous. Please fix it.

Reply:
Done c852b21